compliance/collection/main.go

```diff
@@ -278,15 +284,13 @@ func main() {
 
 	stoppedSig := concurrency.NewSignal()
 
-	go manageReceiveStream(ctx, cli, &stoppedSig)
-
 	if features.RHCOSNodeScanning.Enabled() {
 		rescanInterval := env.NodeRescanInterval.DurationSetting()
 		cmetrics.ObserveRescanInterval(rescanInterval, getNode())
 		log.Infof("Node Rescan interval: %s", rescanInterval.String())
 		sensorC := make(chan *sensor.MsgFromCompliance)
 		defer close(sensorC)
-		go manageSendingToSensor(ctx, cli, sensorC)
+		go manageStream(ctx, cli, &stoppedSig, sensorC)
```

Comment:
We could move those 3 lines before the `if features.RHCOSNodeScanning.Enabled()` and remove the `else` clause in line 315.